pilot/pkg/serviceregistry/kube/conversion.go

```diff
@@ -112,6 +112,7 @@ func convertService(svc v1.Service, domainSuffix string) *model.Service {
 		LoadBalancingDisabled: loadBalancingDisabled,
 		MeshExternal:          meshExternal,
 		Resolution:            resolution,
+		CreationTime:          svc.CreationTimestamp.Time,
```

Comment:
this is not sufficient. Add the same to service entries, consul/cloudfoundry as well. 
@zachgersh please add this to copilot metadata as well (if it matters)

Reply:
That will need to be addressed by follow-on work.